src/main/starlark/builtins_bzl/bazel/exports.bzl

```diff
@@ -17,7 +17,7 @@
 load("@_builtins//:common/java/java_library.bzl", "java_library")
 load("@_builtins//:common/java/java_plugin.bzl", "java_plugin")
 load("@_builtins//:common/java/proto/java_proto_library.bzl", "java_proto_library")
-load("@_builtins//:common/cc/cc_proto_library.bzl", "cc_proto_library")
+load("@_builtins//:common/cc/cc_proto_library.bzl", "cc_proto_aspect", "cc_proto_library")
 
 exported_toplevels = {}
```

Comment:
Here is missing ` "cc_proto_aspect": cc_proto_aspect,` @buildbreaker2021

Reply:
Maybe just adding this back without https://github.com/bazelbuild/bazel/commit/1b1922a9cd5d33f7c50461235b55cffa85f4ac90 is enough?

Reply:
Please address this.